pkg/scheduler/framework/plugins/clusteraffinity/types.go

```diff
@@ -245,6 +245,16 @@ func interpolateWeightFor(cluster *clusterv1beta1.MemberCluster, property string
 		return 0, fmt.Errorf("cannot interpolate weight, observed value %v, observed min %v, observed max %v", f, minF, maxF)
 	}
 
+	if minF == maxF {
```

Comment:
is there a test case for this?

Reply:
Hi Ryan! I have moved some UTs from another PR to this one for review purposes. The UTs for `types.go` were not checked in earlier 😣